choreolib/src/main/java/choreo/trajectory/SwerveSample.java

```diff
@@ -67,9 +67,9 @@ public class SwerveSample implements TrajectorySample<SwerveSample> {
    * @param vx The velocity of the sample in the X direction in m/s.
    * @param vy The velocity of the sample in the Y direction in m/s.
    * @param omega The angular velocity of the sample in rad/s.
-   * @param ax The acceleration of the sample in the X direction in m/s².
-   * @param ay The acceleration of the sample in the Y direction in m/s².
-   * @param alpha The angular acceleration of the sample in rad/s².
+   * @param ax The acceleration of the sample in the X direction in m/s^2.
+   * @param ay The acceleration of the sample in the Y direction in m/s^2.
+   * @param alpha The angular acceleration of the sample in rad/s^2.
```

Comment:
Revert

Reply:
Javadoc was complaining about the non ascii symbol idk

Reply:
Javadoc should be compiled with UTF8

Reply:
Yea, we pass the proper flags to make UTF-8 characters work.
```groovy
    options.addStringOption("charset", "utf-8")
    options.addStringOption("docencoding", "utf-8")
    options.addStringOption("encoding", "utf-8")
```